cmds/shutdown/shutdown.go

```diff
@@ -47,17 +65,44 @@ func main() {
 		op = flag.Args()[0]
 	}
 
-	f, ok := opcodes[op]
+	if op == "" {
+		if flags.h && flags.r {
+			return 0, errors.New("Invalid aruments")
```

Comment:
can we be a little more specific? 
errors.New("both halt and reboot are set and only one can be set")

Reply:
Sure.